packages/eslint-config-finn/es6.js

```diff
@@ -1,6 +0,0 @@
-'use strict';
```

Comment:
Super breaking probably, but spt already did this, so `eslint-config-spt/env-es6` was already included...